docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -0,0 +1,276 @@
+== Feature Management
+
+Feature Management provide a way for Spring Boot applications dynamically access content. Feature Management has a verity of functions, such as Feature Flags that can enable or disable content, Feature Flags for targeting when content is shown, and Feature Gates for dynamically enabling endpoints.
+
+By default, loading of Feature Flags from Azure App Configuration is disabled. And is enabled through configuration:
```

Comment:
This makes it sound like that our customers normally do not use feature flags/this is an "out of the way" feature. Could we phrase this differently? 
Something like "To use feature flags, enable them through configuration:"